catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/receive/SQSMessageReceiver.scala

```diff
@@ -68,14 +68,17 @@ class SQSMessageReceiver @Inject()(
   ) = {
     sourceMetadata.sourceName match {
       case "miro" =>
-        S3ObjectStore.get[MiroTransformable](s3Client, s3Config.bucketName)(
-          hybridRecord.s3key)
+        S3TypedObjectStore.get[MiroTransformable](
+          s3Client,
+          s3Config.bucketName)(hybridRecord.s3key)
       case "calm" =>
-        S3ObjectStore.get[CalmTransformable](s3Client, s3Config.bucketName)(
-          hybridRecord.s3key)
+        S3TypedObjectStore.get[CalmTransformable](
+          s3Client,
+          s3Config.bucketName)(hybridRecord.s3key)
       case "sierra" =>
-        S3ObjectStore.get[SierraTransformable](s3Client, s3Config.bucketName)(
-          hybridRecord.s3key)
+        S3TypedObjectStore.get[SierraTransformable](
+          s3Client,
+          s3Config.bucketName)(hybridRecord.s3key)
```

Comment:
Interesting to note: because of the way we’ve structured this, we’ll be adding another case for METS, which can construct and call an entirely different S3Store. Go past us!